### ISSUE TITLE
rename master to main

### ISSUE THREAD

**Issue by levino** [jsonresume/jsonresume.org#11]
I politely suggest to rename the `master` branch to `main`. Everybody does it these days, it is the new default of github (because BLM, I think).

**Comment by thomasdavis** [comment]
Yeah, as the main contributor over years, I do not want to rewrite that muscle memory.

**Comment by levino** [comment]
Your call, I wont argue. But as a remark: I recently switched to oh my zsh and the git addon. So with `gl`, `gcb feature/fix-1 origin/main`, `gaa`, `gca`, `gp` etc., I rarely type `master` or `main`. So if you switch to oh my zsh at the same time, it might take the sting away :)
